components/KebabPopUp.tsx

```diff
@@ -26,6 +27,7 @@ export const KebabPopUp: React.FC<KebabPopUpProps> = (props) => {
         onBackdropPress={props.onDismiss}
         overlayStyle={Theme.KebabPopUpStyles.kebabPopUp}>
         <Row style={Theme.KebabPopUpStyles.kebabHeaderStyle}>
+          <View></View>
```

Comment:
Why this empty view?

Reply:
Here I used **View, Text** and **Icon** in the row. Then I can able to separate the elements with 
"**justify-content: space-between**". This aligns the header elements of the kebab pop-up.

Reply:
Why we need empty view to make header in the center. We should update text style to make it center.
For kebab popup, I don't see any icon in header coming in near future.